docs/release-notes/self-hosted/self-hosted-v8.0.0.md

```diff
@@ -0,0 +1,105 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Self-hosted v8.0.0.
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/5.5.6
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/5.7.38
+---
+
+# Self-hosted v8.0.0
+
+These release notes are for [Codacy Self-hosted v8.0.0](https://github.com/codacy/chart/releases/tag/8.0.0){: target="_blank"}, released on May 11, 2022. <!-- TODO Update release date -->
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+<!--TODO Check these issues manually
+
+Jira issues with disabled release notes
+-   https://codacy.atlassian.net/browse/CY-5932
+-->
+
+## Upgrading Codacy Self-hosted
+
+Follow the steps below to upgrade to Codacy Self-hosted v8.0.0:
+
+1.  Check the [release notes for all Codacy Self-hosted versions](../index.md#self-hosted) **between your current version and the most recent version** for breaking changes and follow the instructions provided <span class="skip-vale">carefully</span>.
+
+1.  Follow the instructions to [upgrade your Codacy Self-hosted instance](https://docs.codacy.com/v7.1/chart/maintenance/upgrade/).
+
+1.  Update your Codacy command-line tools to the versions with the Git tag `self-hosted-8.0.0`:
+
+    -   [Codacy Analysis CLI](https://github.com/codacy/codacy-analysis-cli/releases/tag/self-hosted-8.0.0)
+    -   [Codacy Coverage Reporter](https://github.com/codacy/codacy-coverage-reporter/releases/tag/self-hosted-8.0.0)
+
+## Breaking changes
+
+ESLint 8 will be enabled by default on new repositories starting on this version of Codacy Self-hosted.
+
+The previous Codacy Self-hosted version [already included ESLint 8 as a new supported tool and deprecated ESLint 7](self-hosted-v7.0.0.md#product-enhancements), and Codacy recommends that you migrate to the new version of the tool to benefit from the new features and fixes of ESLint.
+
+See [how to migrate your configuration files to use ESLint 8](../cloud/cloud-2022-03-31-adding-eslint8.md#migrating-your-configuration-files-to-use-eslint-8). (CY-5848)
```

Comment:
Please review the breaking change for enabling ESLint 8 by default on new repositories.

Reply:
Anything else that you would add @lolgab ? Other than pointing to those docs?